bindings_generator/src/api.rs

```diff
@@ -82,6 +83,10 @@ impl GodotClass {
     pub fn is_pointer_safe(&self) -> bool {
         self.is_refcounted() || self.singleton
     }
+
+    pub fn is_getter(&self, name: &str) -> bool {
+        self.properties.iter().find(|p| p.getter == name).is_some()
+    }
```

Comment:
I know that premature optimization is evil, but this will be done a *lot* of times. Would like to see some profiling results to determine whether this is significant to the compile times.

Reply:
I ran `cargo clean -p gdnative-core -p gdnative-bindings` followed by `time cargo build` to get some cheap timings. 

previous code (in seconds): 
25.26, 25.69, 25.07

new code: 
25.63, 26.16, 25.37, 24.35

There is a bit more, it does seem to be less than a second.

Reply:
That seems fine for now, but small differences like this add up over time. We'd like to watch these timings closely going forward.